<launch>
    <node pkg="pr2_collision_monitor" type="fingertip_monitor" name="r_fingertip_monitor" output="screen" respawn="true">
        <param name="arm" type="str" value="r" />
        <param name="sensor_threshold" type="double" value="2.0" />
    </node>
    <node pkg="pr2_collision_monitor" type="fingertip_monitor" name="l_fingertip_monitor" output="screen" respawn="true">
        <param name="arm" type="str" value="l" />
        <param name="sensor_threshold" type="double" value="2.0" />
    </node>

</launch>
